esmvaltool/diag_scripts/validation.py

```diff
@@ -46,6 +73,15 @@ def save_plotted_cubes(cube, cfg, plot_name):
             save_name = plot_name.replace("png", "nc")
             save_path = os.path.join(cfg['work_dir'], save_name)
             iris.save(cube, save_path)
+            parsed_file_name = plot_name.replace(".png", "").replace("_", " ")
+            prov_name = "Output file: " + parsed_file_name
+            # files are overwritten once; provenance fails second time
```

Comment:
Why bother writing the files the first time round if you're going to overwrite them anyway?

Reply:
depends if the user wants to write files for all the sub-diags, if they want to write only for some then those don't get overwritten (I think - I was too lazy to check proper, I don't think we even need to write any netcdf output if the plots get done well - this is not an actual diagnostic recipe (recipes, with the CMIP5 one too) but rather a tool, utility for monitoring/comparisons)